projects/sidechain-connector-service-typescript/src/types/web3.ts

```diff
@@ -0,0 +1,401 @@
+import { BigNumber } from "bignumber.js"; // TODO change to BN
```

Comment:
Where is it from? AFAIK, there should be official types soon, but why not to use something like https://github.com/slockit/web3.ts for now (there are others as well)?